plugins/org.jkiss.dbeaver.ui.dashboard/src/org/jkiss/dbeaver/ui/dashboard/internal/UIDashboardActivator.java

```diff
@@ -41,13 +43,17 @@ public void start(BundleContext context) throws Exception {
         plugin = this;
         preferences = new BundlePreferenceStore(getBundle());
 
-        DashboardUpdateJob.startUpdating();
+        dashboardUpdateJob = DashboardUpdateJob.startUpdating();
     }
 
     @Override
     public void stop(BundleContext context) throws Exception {
         plugin = null;
         super.stop(context);
+        if (null != dashboardUpdateJob) {
```

Comment:
This doesn't apply to Java because there are no implicit conversions from `null` to `boolean`.

```suggestion
        if (dashboardUpdateJob != null) {
```